Meta/Lagom/Tools/CodeGenerators/JSSpecCompiler/Compiler/Passes/CFGSimplificationPass.cpp

```diff
@@ -7,6 +7,7 @@
 #include "Compiler/Passes/CFGSimplificationPass.h"
 #include "AST/AST.h"
 #include "Function.h"
+#include <AK/TypeCasts.h>
```

Comment:
The "conventional" order of includes in JSSC is libraries, blank line, JSSC stuff.

Reply:
Hmm. That's not a convention we use anywhere else.